service/history/workflow/mutable_state_impl.go

```diff
@@ -3912,13 +3911,13 @@ func (ms *MutableStateImpl) ApplyWorkflowExecutionUpdateAdmittedEvent(event *his
 		return serviceerror.NewInternal("wrong event type in call to ApplyWorkflowExecutionUpdateAdmittedEvent")
 	}
 	if ms.executionInfo.UpdateInfos == nil {
-		ms.executionInfo.UpdateInfos = make(map[string]*updatespb.UpdateInfo, 1)
+		ms.executionInfo.UpdateInfos = make(map[string]*persistencespb.UpdateInfo, 1)
 	}
 	updateID := attrs.GetRequest().GetMeta().GetUpdateId()
-	admission := &updatespb.UpdateInfo_Admission{
-		Admission: &updatespb.AdmissionInfo{
-			Location: &updatespb.AdmissionInfo_HistoryPointer_{
-				HistoryPointer: &updatespb.AdmissionInfo_HistoryPointer{
+	admission := &persistencespb.UpdateInfo_Admission{
+		Admission: &persistencespb.AdmissionInfo{
```

Comment:
Can you rename `*Info` to `Update*Info`, i.e. `UpdateAdmissionInfo`?

Reply:
👍  Done.